include/clang/AST/Expr.h

```diff
@@ -4505,8 +4505,8 @@ class GenericSelectionExpr : public Expr {
 
 /// \brief Represents a Checked C bounds expression.
 class BoundsExpr : public Expr {
-private:
```

Comment:
Why was this explicit access label removed?  I know that members of `class`es default to being `private`, but I find it's always cleaner to be explicit, nonetheless.